wiki/Tournaments/NT/NWT_2019/en.md

```diff
@@ -0,0 +1,267 @@
+| Streamers | ![][flag_US] [Dios Dong](https://osu.ppy.sh/users/3958619), ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519), ![][flag_VN] [steve\_04\_](https://osu.ppy.sh/users/10852911), ![][flag_DE] [WitchOfFrost](https://osu.ppy.sh/users/7511840) |
+| Commentators | ![][flag_US] [Dios Dong](https://osu.ppy.sh/users/3958619), ![][flag_DE] [hi-def](https://osu.ppy.sh/users/10697191), ![][flag_NL] [Just Lucan](https://osu.ppy.sh/users/5007943), ![][flag_DE] [WitchOfFrost](https://osu.ppy.sh/users/7511840), ![][flag_US] [YoshiLover456](https://osu.ppy.sh/users/6843383) |
+| Designers | ![][flag_US] [\_p0ke\_](https://osu.ppy.sh/users/5434711), ![][flag_US] [Cindelluna](https://osu.ppy.sh/users/3039847) |
+| Statistician |  ![][flag_NL] [nikolomara](https://osu.ppy.sh/users/10077264) |
+| Wiki | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/842334)
+- [NT Discord server](https://discord.gg/wrwZWRS)
+- [Livestream](https://www.twitch.tv/niks_tournaments)
+- [NWT 2019 Challonge brackets](https://challonge.com/NWT2019)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vSaXkpCBB_gcPcEEOq93pmLHiOTUf_3n1LYju7NbGF61h5QK19CTuuP2BFBdaOPL-d2UWb06i6iD-wD/pubhtml)**
+
+---
+
+## Participants
+
+| Team | Members |
+| :-- | :-- |
+
+---
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![][flag_US] [Fish-](https://osu.ppy.sh/users/3318654) |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![][flag_AU] [loveleft](https://osu.ppy.sh/users/9240047) |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | ![][flag_DE] [\[Lucky\]](https://osu.ppy.sh/users/1303685) |
+
+---
+
+## Mappools
+
+### Finals
+
+**[Download the mappack here! (123 MB)](https://www.mediafire.com/file/9835f3t5z8xa3a0/Grand+Finals+Pool.zip)**
```

Comment:
Is this supposed to be map pack instead of mappack?

Reply:
Check other tournaments, it's really inconsistent atm.